api-report/runtime-definitions.api.md

```diff
@@ -310,6 +310,16 @@ export interface IInboundSignalMessage extends ISignalMessage {
     type: string;
 }
 
+// @public
+export interface IIncrementalContext {
+    // (undocumented)
+    parentPath: string;
```

Comment:
Maybe summaryPath? I'm not sure "parent" is applicable here, I assume this path includes ID of DDS as well, so it's DDS path